applications/ChatGPT/chatgpt/models/loss.py

```diff
@@ -57,13 +57,9 @@ def __init__(self, clip_eps: float = 0.4) -> None:
 
     def forward(self,
                 values: torch.Tensor,
-                old_values: torch.Tensor,
                 reward: torch.Tensor,
                 action_mask: Optional[torch.Tensor] = None) -> torch.Tensor:
-        values_clipped = old_values + (values - old_values).clamp(-self.clip_eps, self.clip_eps)
```

Comment:
@wenjunyang do you know why do we need to clip the value loss? and why are we taking the max of surr1 and surr2?

Reply:
nvm got some answers here:
https://github.com/openai/baselines/blob/master/baselines/ppo2/model.py#L69-L75
https://github.com/openai/baselines/issues/91

Reply:
thank you. it very helpful.